kiwi/package_manager/apt.py

```diff
@@ -0,0 +1,260 @@
+        information is not used in this case
+        """
+        if not self.distribution:
+            raise KiwiRuntimeError(
+                'No main distribution repository is configured'
+            )
+        bootstrap_script = '/usr/share/debootstrap/scripts/' + \
+            self.distribution
+        if not os.path.exists(bootstrap_script):
+            raise KiwiRequestError(
+                'debootstrap script for %s distribution not found' %
+                self.distribution
+            )
+        bootstrap_dir = self.root_dir + '.debootstrap'
+        if 'apt-get' in self.package_requests:
+            # debootstrap takes care to install apt-get
+            self.package_requests.remove('apt-get')
+        try:
+            dev_mount = MountManager(
+                device='/dev', mountpoint=self.root_dir + '/dev'
+            )
+            dev_mount.umount()
+            Command.run(
+                [
+                    'debootstrap', '--no-check-gpg', self.distribution,
+                    bootstrap_dir, self.distribution_path
+                ], self.command_env
+            )
+            data = DataSync(
+                bootstrap_dir + '/', self.root_dir
+            )
+            data.sync_data(
+                options=['-a', '-H', '-X', '-A']
+            )
+        except Exception as e:
+            Path.wipe(bootstrap_dir)
+            raise KiwiRequestError(
+                '%s: %s' % (type(e).__name__, format(e))
+            )
+        chroot_apt_get_args = self.root_bind.move_to_root(
+            self.apt_get_args
+        )
+        bash_command = [
+            'rm', '-r', '-f', bootstrap_dir, '&&',
+            'chroot', self.root_dir, 'apt-get'
+        ] + chroot_apt_get_args + self.custom_args + [
+            'update'
+        ]
+        Command.run(
+            ['bash', '-c', ' '.join(bash_command)], self.command_env
+        )
+        return self.process_install_requests()
+
+    def process_install_requests(self):
+        """
+        Process package install requests for image phase (chroot)
+        """
+        chroot_apt_get_args = self.root_bind.move_to_root(
+            self.apt_get_args
+        )
+        return Command.call(
+            [
+                'chroot', self.root_dir, 'apt-get'
+            ] + chroot_apt_get_args + self.custom_args + [
+                'install'
+            ] + self.__package_requests(), self.command_env
+        )
+
+    def process_delete_requests(self, force=False):
+        """
+        Process package delete requests (chroot)
+
+        :param bool force: unused
+        """
+        chroot_apt_get_args = self.root_bind.move_to_root(
+            self.apt_get_args
+        )
+        return Command.call(
+            [
+                'chroot', self.root_dir, 'apt-get'
+            ] + chroot_apt_get_args + self.custom_args + [
+                'remove'
+            ] + self.__package_requests(), self.command_env
+        )
```

Comment:
I would write it more readable.

```python
def process_delete_requests(self, force=False):
    """
    Process package delete requests (chroot)

    :param bool force: unused
    """
    cmd = ['chroot', self.root_dir, 'apt-get']
    cmd.extend(self.root_bind.move_to_root(self.apt_get_args))
    cmd.extend(self.custom_args)
    cmd.append('remove')
    cmd.extend(self.__package_requests())

    return Command.call(cmd, self.command_env)
```

And yes. Why param `force` is unused? :wink: